routers/api/v1/repo/find_files.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"code.gitea.io/gitea/modules/base"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/git"
+
+	"github.com/go-enry/go-enry/v2"
+)
+
+var excludedDirs = []string{
```

Comment:
Do we really need hardcoded exclude paths?

Reply:
Agree to remove the hard-coded paths